src/components/Dashboard/FirmsAddressList.vue

```diff
@@ -133,7 +133,10 @@ export default class FirmsAddressList extends Mixins(CommonMixin, CountriesProvi
   @Getter getBusinessAddress!: OfficeAddressIF
 
   // Business
-  get businessAddress (): OfficeAddressIF {
+  get businessAddress (): OfficeAddressIF|null {
+    if (this.getBusinessAddress && this.getBusinessAddress.deliveryAddress &&
+        Object.values(this.getBusinessAddress.deliveryAddress).length === 0) return null
```

Comment:
`Object.values()` should only be used in exceptional circumstances. What are you checking for here? Show me a sample object that this matches.

Also, could any of this this be simplified with [isEmpty](https://www.geeksforgeeks.org/lodash-_-isempty-method/)?

Reply:
`this.getBusinessAddress` will be one of the four samples:
`this.getBusinessAddress = null` (case 1)
`this.getBusinessAddress = { deliveryAddress: null }` (case 2)
`this.getBusinessAddress = { deliveryAddress: {} }` (case 3)
`this.getBusinessAddress = { deliveryAddress: { actions ... }` (case 4, correct data)

I am using `Object.values()` for the case 3.

Reply:
Thanks for the scenario. Very helpful.

I counsel against using Object.values(). I think you should use `isEmpty()` for case 3.

Reply:
I agree. I changed it to isEmpty() and it's very powerful.